inloop/grading/admin.py

```diff
@@ -1,6 +1,66 @@
+
 from django.contrib import admin
+from django.http import HttpResponse
 
 from inloop.grading.models import DetectedPlagiarism, PlagiarismTest
 
-admin.site.register(DetectedPlagiarism)
-admin.site.register(PlagiarismTest)
+
+def download_jplag_zip(modeladmin, request, queryset):
+    """Fetch the stored jplag zip file and return it as a download"""
+    if queryset.count() != 1:
+        modeladmin.message_user(
+            request, "You can not download more or less than one zip file at a time."
+        )
+        return
+    test = queryset[0]
+    if not test.zip_file:
+        modeladmin.message_user(
+            request, "The selected plagiarism test has no downloadable zip file."
+        )
+    response = HttpResponse(open(test.zip_file.path, 'rb'), content_type='application/zip')
```

Comment:
Note to myself: in the long run, we cannot leave this, because this file object is not closed. This is related to #186, where I also need to implement a proper and secure mechanism to download (or display) uploaded files.

Reply:
I changed this to 
```python
with open(test.zip_file.path, 'rb') as source:
    response = HttpResponse(source, content_type='application/zip')
```
Now the file will be closed automatically.